app/app.go

```diff
@@ -372,7 +347,7 @@ func init() {
 func NewDesmosApp(
 	logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bool, skipUpgradeHeights map[int64]bool,
 	homePath string, invCheckPeriod uint, encodingConfig simappparams.EncodingConfig,
-	appOpts servertypes.AppOptions, baseAppOptions ...func(*baseapp.BaseApp),
+	appOpts servertypes.AppOptions, wasmEnabledProposals []wasm.ProposalType, baseAppOptions ...func(*baseapp.BaseApp),
```

Comment:
Add `wasmEnabledProposals` parameter since wasm sim tests requires enabling all proposal types like:
https://github.com/CosmWasm/wasmd/blob/v0.30.0/app/sim_test.go#L155